docs/example.md

```diff
@@ -0,0 +1,115 @@
+# Full workflow tutorial
+
+This example demonstrates how to export application deployed on Openshift,
+to Nulecule specification and then import that application to Kubernetes.
+
+This is a 3 step tutorial:
+    1. Deploy a sample application on Openshift.
+    2. Export the application using openshift2nulecule.
+    3. Import the exported artifacts into Kubernetes using Atomic App.
+
+
+This has been tested on ADB 1.7.1 as Fedora 23 on local machine
+
+## Deploy Docker registry on your local machine
+ 1. Run Docker registry on you local machine.
+    - `docker run -d -p 5000:5000 --name registry registry:2`
+
+
+## Start ADB and deploy sample OpenShift application
+ 1. Clone the ADB repository
+
+    `git clone https://github.com/projectatomic/adb-atomic-developer-bundle.git`
+
+ 1. Run OpenShift instance in ADB
+
+    - `cd adb-atomic-developer-bundle/components/centos/centos-openshift-setup`
+    - `vagrant up`
+    - `vagrant ssh`
+
+ 1. Add Docker Registry that runs on your local machine as insecure registry (in ADB box)
+     
+    - `sudo vi /etc/sysconfig/docker`
+    - Uncommend INSECURE_REGISTRY line and add your local Docker Registry (`'--insecure-registry=10.1.2.1:5000'`)
```

Comment:
@kadel I found that this line is already uncommented, so why not rephrase this as:

Add your local machine's Docker registry (which was started in the first step) by appending `--insecure-registry 10.1.2.1:5000` to `INSECURE_REGISTRY`